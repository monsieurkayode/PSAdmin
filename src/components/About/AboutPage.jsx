import React from 'react';
import { Link } from 'react-router-dom';

/**
 *
 * @function AboutPage
 *
 * @returns {JSX}
 */
const AboutPage = () => (
  <div className="container jumbotron">
    <div className="container-fluid">
      <h1>About PSAdmin</h1>
      <p>
        This application uses React, Redux, React-Router and a variety
        of other helpful libraries.
      </p>
      <Link to="/about">
        <button type="button" className="btn btn-lg btn-primary">
          Learn more
        </button>
      </Link>
    </div>
  </div>
);

export default AboutPage;
