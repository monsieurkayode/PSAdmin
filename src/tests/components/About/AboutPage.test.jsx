import React from 'react';
import { shallow } from 'enzyme';
import AboutPage from '../../../components/About/AboutPage';

describe('AboutPage Component', () => {
  const wrapper = shallow(<AboutPage />);

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
