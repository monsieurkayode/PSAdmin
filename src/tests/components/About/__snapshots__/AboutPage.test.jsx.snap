// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutPage Component renders correctly 1`] = `
<div
  className="container jumbotron"
>
  <div
    className="container-fluid"
  >
    <h1>
      About PSAdmin
    </h1>
    <p>
      This application uses React, Redux, React-Router and a variety of other helpful libraries.
    </p>
    <Link
      replace={false}
      to="/about"
    >
      <button
        className="btn btn-lg btn-primary"
        type="button"
      >
        Learn more
      </button>
    </Link>
  </div>
</div>
`;
